executor/builder.go

```diff
@@ -38,13 +38,14 @@ import (
 // executorBuilder builds an Executor from a Plan.
 // The InfoSchema must not change during execution.
 type executorBuilder struct {
-	ctx context.Context
-	is  infoschema.InfoSchema
+	ctx      context.Context
+	is       infoschema.InfoSchema
+	priority int
```

Comment:
I've thought it, but I found that StatementContext just store some SQL related things currently, so it maybe not appropriate.

I also thought other places:
1. Transaction option.... not appropriate because this may happen:

```
begin
cheap query
expensive query
commit
```

We'll do txn.SetOption() several times contradictory.

2. Session variable.

Obviously it's not a good place.

3. StatementContext

Also not appropriate.

4. TableScan & IndexScan related Executor

Finally I chose to put it here, because distsql just use them.